Client/Ecosia/Experiments/Unleash/EngagementServiceFeature.swift

```diff
@@ -5,25 +5,25 @@
 import Foundation
 import Core
 
-struct APNConsentUIExperiment {
+struct EngagementServiceFeature {
```

Comment:
Any reason not to call this `EngagementServiceExperiment`? Or even `BrazeExperiment`? Just asking since that was the suffix we used for the other experiments inside this folder.

Reply:
Good question @lucaschifino .
Basically, just this morning, the [Braze Implementation](https://ecosia.atlassian.net/wiki/spaces/MOB/pages/2826600554/Braze+Implementation) doc was moved under "Features". I then asked for clarifications as we were going thru applying some changes anyway.
Realised the following afterwards:
- The "experiment" is still in place and will consist of checking whether opt in rates are up to industry standards
- Braze, regardless of the outcome, is here to stay as a **Feature**

Reply:
Got it, I do feel that code-wise this is still an experiment and that this class here will be deleted once we move out of the experiment and it becomes a proper feature, so I would still vote for naming `BrazeExperiment`.

As I said not a blocker though and I guess since it was already merged that's how it will be for now 😅

Reply:
I see your point and the folder references not matching the concrete intention of this one.
The flag itself though will remain as per the core-markets segment, so this file will still be used but I guess a some of the other stuff such as "variants" will be removed/modified.